config/routes.rb

```diff
@@ -1,59 +1,14 @@
-require 'sidekiq/web'
 Rails.application.routes.draw do
-  use_doorkeeper do
-    controllers tokens: 'tokens'
-
-    skip_controllers :authorizations, :applications, :authorized_applications
-  end
-
-  devise_for :users
-  mount Rswag::Ui::Engine => '/api-docs'
-  mount Rswag::Api::Engine => '/api-docs'
-  namespace :v1 do
-    resources :me, only: [:index] do
-    end
-  end
-
-  namespace :api, defaults: { format: :json } do
-    resources :force_update_app_versions, only: [:index] do
-    end
-
-    resources :users_verify_confirmation_token, only: [:create] do
-    end
-
-    resources :users_passwords, only: [:create] do
-    end
-
-    resources :users_registrations, only: [:create] do
-    end
-
-    resources :users_verify_reset_password_requests, only: [:create] do
-    end
-
-    resources :users_reset_password_requests, only: [:create] do
-    end
-
-    resources :messages, only: [:create] do
-    end
-
-    resources :chanels, only: %i[index show destroy] do
-      resources :messages, only: %i[index destroy] do
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:show, :update] do
+        collection do
+          post 'register'
+          post 'confirm'
+        end
```

Comment:
The `config/routes.rb` file has been modified to remove several routes. Ensure that these routes are not being used by the frontend or other services before removing them to avoid breaking the application.